test/header_parser_tests/expected_bindings/_expected_packed_structs_bindings.dart

```diff
@@ -0,0 +1,52 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+import 'dart:ffi' as ffi;
+
+class NormalStruct1 extends ffi.Struct {
+  @ffi.Int8()
+  external int a;
+}
+
+/// Should not be packed.
+class StructWithAttr extends ffi.Struct {
+  external ffi.Pointer<ffi.Int32> a;
+
+  external ffi.Pointer<ffi.Int32> b;
+}
+
+/// Should be packed with 1.
+@ffi.Packed(1)
+class PackedAttr extends ffi.Struct {
+  @ffi.Int32()
+  external int a;
+}
+
+/// Should be packed with 8.
+@ffi.Packed(8)
+class PackedAttrAlign8 extends ffi.Struct {
+  @ffi.Int32()
+  external int a;
+}
+
+/// Should be packed with 2.
+@ffi.Packed(2)
+class Pack2WithPragma extends ffi.Struct {
+  @ffi.Int32()
+  external int a;
+}
+
+/// Should be packed with 8.
```

Comment:
?

Reply:
Turns out that #pragma pack(X) sets the Upper bound value of the alignment. (While the alignment(X) attribute sets the lower bound.) I have updated the test accordingly.